osu.Game/Overlays/Profile/ProfileHeader.cs

```diff
@@ -316,18 +316,18 @@ public User User
 
         private void loadUser()
         {
-            coverContainer.Add(new AsyncLoadWrapper(new UserCoverBackground(user)
+            LoadComponentAsync(new UserCoverBackground(user)
             {
                 RelativeSizeAxes = Axes.Both,
                 Anchor = Anchor.Centre,
                 Origin = Anchor.Centre,
                 FillMode = FillMode.Fill,
                 OnLoadComplete = d => d.FadeInFromZero(200)
-            })
+            },
+            ucb =>

```

Comment:
just make this `coverContainer.Add` instead of a lambda method. move the depth assignment to the object assignment above.

Reply:
Oh, okay.. will do, thought that was the only way to do it.